packages/node_modules/pouchdb-replication/src/index.js

```diff
@@ -5,23 +5,25 @@ function replication(PouchDB) {
   PouchDB.replicate = replicate;
   PouchDB.sync = sync;
 
-  Object.defineProperty(PouchDB.prototype, 'replicate', {
-    get: function () {
-      var self = this;
-      return {
-        from: function (other, opts, callback) {
-          return self.constructor.replicate(other, self, opts, callback);
-        },
-        to: function (other, opts, callback) {
-          return self.constructor.replicate(self, other, opts, callback);
-        }
-      };
-    }
-  });
+  if (!PouchDB.prototype.replicate) {
```

Comment:
Or do i need to check with  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptor ?